### DISCUSSION TITLE
VERSION_MINOR in version.py

### DISCUSSION THREAD

**Discussion by cremesk** [searx/searx#3330] [General]
it looks like it was forgotten to raise the version number in:
https://github.com/searx/searx/blob/master/searx/version.py#L21

**Comment by cremesk** [comment]
fixed now in: https://github.com/searx/searx/commit/ee7d173c5afcc3a3848692234fbd4aee06a56dbf

**Reply by kvch** [reply]
Thank you for reporting it!
